### ISSUE TITLE
Allow products to be excluded from ticket-generation - 2nd incarnation

### ISSUE THREAD

**Issue by pc-coholic** [pretix/pretix#548]
Sorry for warming up #456 again...

I was actually happy with the solution we went with, as it looked good in theory. But in practice, it's not as good...

As it turns out, the "is Admission"-property is used in the Dashboard-tiles "Attendees (ordered)" and "Attendees (payed)".

So right now, we can either choose to
- screw up the numbers in the statistics by declaring non-admission products like T-Shirts as admission-items
- or not to touch the statistics and have tickets generated for all items - even those we might not absolutely need a ticket for.

Real life example: SHA has admission-tickets, T-Shirts and Camping-stuff (like Chairs, Tents, etc.). In order to not screw up the Attendees-statistics on the dashboard, we opted to generate tickets for all items. This brings us to a point, where certain groups now have 130 pages worth of tickets for chairs - even though they don't actually need to print and bring them, as camping-equipment is not checked against a paper-ticket.

Long story short: I would like to advocate for a "generate ticket for this item"-property on items.

Migration path:
```
if (generateTicketsForAllItems == true):
  Items.all.setGenerateTicket = true
else:
  Items.admission.setGenerateTicket = true
```

**Comment by pc-coholic** [comment]
Fixed in 2aa246b3d534edf625302660b0894241b1a727e8
